docs/interactions/Slash_Commands.md

```diff
@@ -742,6 +782,31 @@ Edits a followup message for an Interaction. Functions the same as [Edit Webhook
 
 Deletes a followup message for an Interaction. Returns `204` on success.
 
+## Get Guild Application Command Permissions % GET /applications/{application.id#DOCS_TOPICS_OAUTH2/application-object}/guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/commands/permissions
+
+Fetches command permissions for all commands for your application in a guild. Returns an array of [ApplicationCommandPermissions](#DOCS_INTERACTIONS_SLASH_COMMANDS/applicationcommandpermissions).
```

Comment:
How am I supposed to correlate the permissions and the command they belong to with this route? (if it really does only return an array of `ApplicationCommandPermissions`, which it might not... I didn't check.)